code/game/machinery/vending.dm

```diff
@@ -889,7 +889,8 @@
 		/obj/item/weapon/reagent_containers/syringe = 12,
 		/obj/item/device/healthanalyzer = 5,
 		/obj/item/weapon/reagent_containers/glass/beaker = 4,
-		/obj/item/weapon/reagent_containers/dropper = 2)
+		/obj/item/weapon/reagent_containers/dropper = 2,
+		/obj/item/weapon/reagent_containers/blood/OMinus = 2)

```

Comment:
I question how powerful this will end up being.
That's a lot of blood, especially on maps with more than one vendor.